gamebook/tests/test_chance_option.py

```diff
@@ -0,0 +1,21 @@
+from gamebook.tests.mock_nextscene import nextscene1
+from gamebook.tests.mock_chance_option import option1
+
+"""
+using mock Next_scene nextscene1 and mock option option1
+
+nextscene1:
+    next_scene: scene1
+    success_rate: not set (default is 1.0)
+
+option1:
+    text: hello world
+    keys: h, next
+    nextscene: nextscene1
+"""
+
+def test_initialization():
+    
```

Comment:
I would not use the mock this way. Initialize the object here, so it is easy to see what changes. I had a different idea for the mocks (to create a [`Factory`](https://www.tutorialspoint.com/python_design_patterns/python_design_patterns_factory.htm) that generates a mock easily - but it isn't needed on 2nd thought when the code is so simple.

Reply:
ok i thought that is what you meant, i'll read about factory

Reply:
No need to implement a Factory now, just put the code like before inside it for now. It's ok, sorry if this is misleading - I'm trying so this will go faster so if it isn't totally needed let's push it for later.

Reply:
:+1: